src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div>
+  <div>Octogons shapes go here</div>
```

Comment:
We can remove this `div`. We'll determine how we want to add that detail later

Reply:
fixed